packages/graphql-api/package.json

```diff
@@ -34,7 +37,9 @@
     "node-dev": "^3.1.3"
   },
   "dependencies": {
-    "@accounts/server": "0.1.0-alpha.58382c10",
+    "@accounts/common": "^0.1.0-beta.14",
```

Comment:
We deleted the `@accounts/common` package on the accounts repository, all types are in `@accounts/types`.

Reply:
Resolved: Fixed.